### DISCUSSION TITLE
How does the "`" disappear from the output?

### DISCUSSION THREAD

**Discussion by nilslindemann** [TiddlyWiki/TiddlyWiki5#6253] [Q&A] [answered]
For example, [this](https://github.com/Jermolene/TiddlyWiki5/blob/122de63b71a2454314365ac396e051ad1db77641/core/templates/html-div-tiddler.tid) and [this](https://github.com/Jermolene/TiddlyWiki5/blob/122de63b71a2454314365ac396e051ad1db77641/core/templates/css-tiddler.tid) and [this](https://github.com/Jermolene/TiddlyWiki5/blob/122de63b71a2454314365ac396e051ad1db77641/core/templates/javascript-tiddler.tid) template contains these characters, but they are not translated to code elements, but stripped away from the output. How does this work?

**Comment by dubiouscript** [comment]
:question:  https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/parsers/wikiparser/rules/codeinline.js

**Reply by nilslindemann** [reply]
Yes, I already guessed that this is used. I was searching for how these `<code>...</code>` elements then disappear. See my answer below.

**Comment by nilslindemann** [comment, marked as answer]
I figured it out. It is the ["format" attribute](https://github.com/Jermolene/TiddlyWiki5/blob/122de63b71a2454314365ac396e051ad1db77641/core/modules/widgets/view.js#L55-L91) of the [ViewWidget](https://tiddlywiki.com/#ViewWidget).

[getValueAsPlainWikified()](https://github.com/Jermolene/TiddlyWiki5/blob/122de63b71a2454314365ac396e051ad1db77641/core/modules/widgets/view.js#L148) does the job in this case.

For example, [create two tiddlers](https://tiddlywiki.com/#A%20Strange%20Paragraph:%5B%5BA%20Strange%20Paragraph%5D%5D%20%5B%5BViews%2C%20Views%2C%20Views%5D%5D) and set the content of the first to

```
`<` `p` `>`  Hi `<` `/` `p` `>`
```

and the content of the second to

```
{{A Strange Paragraph||$:/core/templates/raw-static-tiddler}} // getValueAsPlainWikified() //

---

{{A Strange Paragraph||$:/core/templates/plain-text-tiddler}}

---

{{A Strange Paragraph||$:/core/templates/html-tiddler}}

---

{{A Strange Paragraph||$:/core/templates/canonical-uri-external-raw}}

---

{{A Strange Paragraph||$:/core/templates/css-tiddler}}

---

{{A Strange Paragraph||$:/core/templates/javascript-tiddler}}

---

{{A Strange Paragraph||$:/core/templates/json-tiddler}}

---

{{A Strange Paragraph||$:/core/templates/tiddler-metadata}}

---

{{A Strange Paragraph||$:/core/templates/single.tiddler.window}}

---

{{A Strange Paragraph||$:/core/templates/static-tiddler}}
```
